lambda/main.go

```diff
@@ -71,14 +71,23 @@ func Handler(ctx context.Context, evt json.RawMessage) (string, error) {
 
 	var b backend.Backend
 	var err error
-	if backendOpt == "statsd" {
+	switch strings.ToLower(backendOpt) {
+	case "statsd":
 		statsdHost := os.Getenv("STATSD_HOST")
 		statsdTags := strings.ToLower(os.Getenv("STATSD_TAGS")) == "true"
```

Comment:
I learnt recently that `strings.EqualFold` is a thing for case-insensitive comparison, but same/same.